lib/static/new-ui/features/suites/components/SuitesTreeView/index.tsx

```diff
@@ -7,16 +7,12 @@ import {
 } from '@gravity-ui/uikit/unstable';
```

Comment:
The essence of changes in this file — expose imperative handle to allow scrolling to certain item in tree and migrate to hooks redux API instead of `connect()`.